components/aws/sagemaker/tests/unit_tests/tests/MonitoringSchedule/test_MonitoringSchedule_component.py

```diff
@@ -0,0 +1,178 @@
+from commonv2.sagemaker_component import SageMakerJobStatus
+
+from unittest.mock import patch, MagicMock
+import unittest
+from MonitoringSchedule.src.MonitoringSchedule_spec import (
+    SageMakerMonitoringScheduleSpec,
+)
+from MonitoringSchedule.src.MonitoringSchedule_component import (
+    SageMakerMonitoringScheduleComponent,
+)
+
+
+class MonitoringScheduleComponentTestCase(unittest.TestCase):
```

Comment:
What is the coverage for the component file.

Reply:
79% for MonitoringSchedule_component.py